packages/backend/server/src/modules/quota/configure/feature.ts

```diff
@@ -0,0 +1,210 @@
+import { Injectable, OnModuleInit } from '@nestjs/common';
+
+import { PrismaService } from '../../../prisma';
+import { migrateNewFeatureTable } from './migration';
+import { CommonFeature, Feature, FeatureKind, FeatureType } from './types';
+
+const Features: Feature[] = [
+  {
+    feature: FeatureType.Feature_EarlyAccess,
+    type: FeatureKind.Feature,
+    version: 1,
+    configs: {
+      whitelist: ['@toeverything.info'],
+    },
+  },
+];
+
+@Injectable()
+export class FeatureService implements OnModuleInit {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async onModuleInit() {
+    // upgrade features from lower version to higher version
+    for (const feature of Features) {
+      await this.upsertFeature(feature);
```

Comment:
can be moved to data-migration script now
`yarn data-migration create`
`yarn data-migration run`